docs/getting-started/default-networking/Demonstration.md

```diff
@@ -86,12 +92,17 @@ On core-02:
     calicoctl endpoint $(calicoctl container workload-D endpoint-id show) profile append PROF_D
     calicoctl endpoint $(calicoctl container workload-E endpoint-id show) profile append PROF_A_C_E
 
-*Note that creating a new profile with `calicoctl profile add` will work on any Calico node, but assigning an endpoint a profile with `calicoctl endpoint <ENDPOINT_ID> profile append` will only work on the Calico node where the container is hosted.*
+*Note that creating a new profile with `calicoctl profile add` will work on 
```

Comment:
Ah, well the calicoctk container command is node specific
 
.. so it's kinda true.